### ISSUE TITLE
Line breaks are treated incorrectly

### ISSUE THREAD

**Issue by chucker** [trenz-gmbh/trenz-docs-api#15]
(This seems to be happening on the API end.)

Given Markdown like:

```markdown
A database will have one or more **tables**, each of which contains
**rows**.
```

TRENZ Docs will currently insert a line break. Azure DevOps doesn't seem to do that. [Neither does GitHub.](https://github.com/ricardoboss/trenz-docs-test/blob/main/docs/line-breaks.md)

**Comment by ricardoboss** [comment]
This seems to be caused by an extension, which isn't even enabled AFAICT: https://github.com/xoofx/markdig/blob/master/src/Markdig.Tests/Specs/HardlineBreakSpecs.md

**Comment by chucker** [comment]
Yeah, that's definitely not the standard Markdown behavior. Line breaks should only be auto-inserted after two spaces. (Which I think is a bit weird, but oh well.)
